GoMeet/src/main/java/com/google/sps/servlets/VoteMeetingTimeServlet.java

```diff
@@ -0,0 +1,99 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
```

Comment:
Why does this one have copyright comments and others don't?

Reply:
Sorry, this was copied over from the other servlet file. Just confirming - should we need to have license comments in _every_ file, or no license comments across all files?

Reply:
I checked with Vivian, since you made the same comment in her PR, and I've removed the license comments.